### PR TITLE
added documentation for using Web Font Loader as CommonJS module

### PR DESCRIPTION
**Pull request by Prinzhorn**
It's not documented anywhere and I had to search for it to come across #226 (and then I had to guess/search the package name on npm too)

To save others some time it should be documented somewhere.

### CODE REVIEW

**Comment by bramstein** [comment]
This is a good idea. One small thing needs to be fixed though, the package name is incorrect.

File: README.md [hunk 1]
```diff
@@ -57,6 +57,18 @@ Using the Web Font Loader asynchronously avoids blocking your page while loading
 
 The FOUT can be more easily avoided when loading the Web Font Loader synchronously, as it will automatically set the `wf-loading` class on the HTML element as soon as `Webfont.load` has been called. The browser will wait for the script to load before continuing to load the rest of the content, FOUT is avoided.
 
+Web Font Loader is also available on npm as a CommonJS module. Just `npm install webfontconfig` and then require it in your code.
```
Comment (bramstein) [inline README.md]: The official package name is `webfontloader`.

**Comment by Prinzhorn** [comment]
I guess that was some copy/paste mistake

**Comment by bramstein** [comment]
I'll merge it in, thanks a lot!

**Pull request merged**